### PR TITLE
Resolved Youtube link issue #25079

### PR DESCRIPTION
**Pull request by hound77**
### Description

Removed Youtube link from index.md file in webAssembly.

### Motivation

As Link was not available, So removed the Youtube video link from documentation.

### CODE REVIEW

File: files/en-us/webassembly/using_the_javascript_api/index.md [hunk 1]
```diff
@@ -78,8 +78,7 @@ In Firefox 54+, the Developer Tool Debugger Panel has functionality to expose th
 
 ![Developer tools debugger panel highlighting a module.](wasm-debug.png)
 
-In addition to viewing WebAssembly as text, developers are able to debug (place breakpoints, inspect the callstack, single-step, etc.) WebAssembly using the text format. See [WebAssembly debugging with Firefox DevTools](https://www.youtube.com/watch?v=R1WtBkMeGds) for a video preview.
-
+In addition to viewing WebAssembly as text, developers are able to debug (place breakpoints, inspect the callstack, single-step, etc.) WebAssembly using the text format.
 ## Memory
```
Comment (dawei-wang) [inline files/en-us/webassembly/using_the_javascript_api/index.md:83]:
```suggestion

## Memory

```

**Review by dawei-wang — COMMENTED** [review]
linting

**Comment by hound77** [comment]
> linting

@dawei-wang okay sure will resolve the linting issue

**Review by estelle — APPROVED** [review]
thank you

**Pull request merged**